### ISSUE TITLE
Allow embedding videos on app store pages

### ISSUE THREAD

**Issue by gerhardt-bergmann** [nextcloud/appstore#1507]
Please allow app developers to include video presentations on the app store pages of their apps.

The text description and the screenshots that can be added currently are useful but in many cases they are not sufficient. People today are very visual and they would like to see a video presentation of the app, so that they can see the app in action and get an idea about its general look and its main features, before they decide if it fits their needs. The video presentation could be a youtube, vimeo, dailymotion, odysee, etc. video embeded on the page using an iframe, like [a video presentation](https://wordpress.org/plugins/duplicator/) on a WordPress plugin page.

Please consider accepting the `<video></video>` tag in the `appinfo/info.xml` files of apps, so that app authors can add short video presentations to their apps. If you think it's necessary you can restrict the source of the iframe video to some specific video platforms, restrict the duration of the video, etc.

Edit Daphne:
Embedding videos is a good idea but we do not accept embedding of Youtube, Vimeo, or other providers who add tracking. We would accept a PR that allows embedding a video in the app store from Peertube. This can be implemented probably directly into the app store and does not require changing the info.xml file.

**Comment by DaphneMuller** [comment]
Hi @gerhardt-bergmann , thank you for your suggestion. It would indeed be nice to embed videos. One consideration is the tracking, we do not want to embed Youtube or Vimeo. We also would not want to host the videos on our infrastructure. 

We would welcome a PR that embeds Peertube videos.

**Comment by gerhardt-bergmann** [comment]
Indeed, rejecting embedding videos from platforms that add tracking, like YouTube and Vimeo, is a good idea. This is why I suggested that the source of the iframed video can be restricted. Yet, the proposed `<video></video>` tags should be included on [this list](https://docs.nextcloud.com/server/latest/developer_manual/app_development/info.html), alongside the other tags (like `<id></id>`, `<name></name>`, `<summary></summary>`, `<description></description>`, etc.), so that each developer knows that he can include the `<video></video>` tags in his `appinfo/info.xml` file, to specify the URL of the video presentation of his app. It should be an optional tag, of course, like many others. Only the developers who need it will use it.
